src/main/resources/application.properties

```diff
@@ -20,6 +20,7 @@ lwgs.searchindex.client.sedex.recipient-id=${SEDEX_RECIPIENT_ID:LWGS-1848-CH}
 lwgs.searchindex.client.sedex.message.type.full-export=${SEDEX_MESSAGE_TYPE_FULL_EXPORT:2518}
 lwgs.searchindex.client.sedex.message.type.incremental=${SEDEX_MESSAGE_TYPE_INCREMENTAL:2518}
 lwgs.searchindex.client.sedex.message.class=0
+lwgs.searchindex.client.sedex.data.processor.version=@lgs.person.data.processor.version@
```

Comment:
Where is the value lgs.person.data.processor.version?

Reply:
pom.xml:25
<lgs.person.data.processor.version>2.0.3</lgs.person.data.processor.version>